server/server.go

```diff
@@ -107,29 +108,35 @@ func (r *rodeServer) BatchCreateOccurrences(ctx context.Context, occurrenceReque
 
 func (r *rodeServer) EvaluatePolicy(ctx context.Context, request *pb.EvaluatePolicyRequest) (*pb.EvaluatePolicyResponse, error) {
 	var err error
-	log := r.logger.Named("EvaluatePolicy").With(zap.String("policy", request.Policy), zap.String("resource", request.ResourceURI))
+	log := r.logger.Named("EvaluatePolicy").With(zap.String("policy", request.Policy), zap.String("resource", request.ResourceUri))
 	log.Debug("evaluate policy request received")
 
-	// check OPA policy has been loaded
-	err = r.opa.InitializePolicy(request.Policy)
+	policy, _ := r.GetPolicy(ctx, &pb.GetPolicyRequest{Id: request.Policy})
+	// check OPA policy has been loaded, using the policy id
+	err = r.opa.InitializePolicy(request.Policy, policy.Policy.RegoContent)
 	if err != nil {
 		log.Error("error checking if policy exists", zap.Error(err))
 		return nil, status.Error(codes.Internal, "check if policy exists failed")
 	}
 
 	// fetch occurrences from grafeas
-	listOccurrencesResponse, err := r.grafeasCommon.ListOccurrences(ctx, &grafeas_proto.ListOccurrencesRequest{Parent: "projects/rode", Filter: fmt.Sprintf(`"resource.uri" == "%s"`, request.ResourceURI)})
+	listOccurrencesResponse, err := r.grafeasCommon.ListOccurrences(ctx, &grafeas_proto.ListOccurrencesRequest{Parent: "projects/rode", Filter: fmt.Sprintf(`"resource.uri" == "%s"`, request.ResourceUri)})
 	if err != nil {
-		log.Error("list occurrences failed", zap.Error(err), zap.String("resource", request.ResourceURI))
+		log.Error("list occurrences failed", zap.Error(err), zap.String("resource", request.ResourceUri))
 		return nil, status.Error(codes.Internal, "list occurrences failed")
 	}
 	log.Debug("Occurrences found", zap.Any("occurrences", listOccurrencesResponse))
 
-	// json encode occurrences. list occurrences response should not generate error
-	input, _ := protojson.Marshal(proto.MessageV2(listOccurrencesResponse))
+	input, _ := json.Marshal(listOccurrencesResponse)
```

Comment:
shouldn't we be using `protojson.Marshal` here?

Reply:
you're right. in a previous commit i wasn't using a proto message so that line wasn't working. I switched back but didn't change the marshaler